### ISSUE TITLE
Constantly get "Undefined index: team_id" error 

### ISSUE THREAD

**Issue by suan** [yannickglt/alfred-slack#7]
I get the following error whenever issuing any slack command through alfred, even when trying to do `slack --token` or `slack --refresh`:

```
[ERROR: alfred.workflow.input.scriptfilter] XML Parse Error 'The operation couldn’t be completed. (NSXMLParserErrorDomain error 4.)'. Row 1, Col 1: 'Document is empty' in XML:
Notice: Undefined index: team_id in /Users/Suan/dotfiles/Alfred.alfredpreferences/workflows/user.workflow.34900165-4E0A-4199-93C4-3FD8B1066633/scripts/helpers/service/MultiTeamSlackService.php on line 66
```

Am running the latest versions of this workflow and Alfred

**Comment by morriswinkler** [comment]
me too

**Comment by yannickglt** [comment]
Should be fixed with v2.1.0.
